spidermon/contrib/scrapy/monitors.py

```diff
@@ -100,6 +103,17 @@ class MyCustomStatMonitor(BaseStatMonitor):
     """
 
     fail_if_stat_missing = True
+    threshold_datatype = float
```

Comment:
Is there any risk on backward compatibility with all the `BaseStatMonitor` already implemented? 
Both in this package and customs that developers may have created.

For example, If I have already defined a threshold vallue as `int`, this feature would make spidermon to read the value as `float` instead. I'm not sure if that may have unexpected side effects

Reply:
I feel like reading an int as a float (in the monitors` use cases) will never cause an issue whilst the opposite wouldn't be true. 
For example: if a threshold is 135 and is read as 135.0, I reckon that wouldn't make a functional difference. If that's the case, what other option do we have? I think reading a float by default is the most backwards compatible thing we can do.

Reply:
My concern is that when testing for equality, there could be some unexpected round-off errors that caused that for some value `n`, `int(n) != float(x)`. 

However, I couldn't find any examples where this may happens.